cob_obstacle_distance/include/cob_obstacle_distance/distance_manager.hpp

```diff
@@ -52,37 +52,39 @@
 #include <tf_conversions/tf_kdl.h>
 #include <tf_conversions/tf_eigen.h>
 
+#include <kdl/chainfksolvervel_recursive.hpp>
+#include <ros/ros.h>
+
 #include "cob_obstacle_distance/marker_shapes.hpp"
 #include "cob_obstacle_distance/shapes_manager.hpp"
-#include "cob_obstacle_distance/chainfk_solvers/advanced_chainfksolverpos_recursive.h"
+#include "cob_obstacle_distance/chainfk_solvers/advanced_chainfksolver_recursive.hpp"
 #include "cob_obstacle_distance/obstacle_distance_data_types.hpp"
 #include "cob_obstacle_distance/Registration.h"
+#include "cob_obstacle_distance/PredictDistance.h"
+
 
-#include <ros/ros.h>
 
 class DistanceManager
 {
     private:
-        typedef std::unordered_map<std::string, ObstacleDistance> t_map_ObstacleDistance;
-        typedef std::unordered_map<std::string, ObstacleDistance>::iterator t_map_ObstacleDistance_iter;
-        typedef std::unordered_map<std::string, ObstacleDistance>::const_iterator t_map_ObstacleDistance_citer;
+        typedef std::unordered_map<std::string, ObstacleDistance> Map_ObstacleDistance_t;
+        typedef std::unordered_map<std::string, ObstacleDistance>::iterator Map_ObstacleDistance_iter_t;
+        typedef std::unordered_map<std::string, ObstacleDistance>::const_iterator Map_ObstacleDistance_citer_t;
```

Comment:
so which `typedef` naming convention will we use, eventually?
See https://github.com/ipa-fxm/cob_control/pull/7/files#diff-57cd559461af923db44d097bb99c7aa8R37